eventsub-common/src/main/java/com/github/twitch4j/eventsub/events/AutomodMessageHoldEvent.java

```diff
@@ -0,0 +1,41 @@
+package com.github.twitch4j.eventsub.events;
+
+import com.github.twitch4j.eventsub.domain.chat.Message;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Setter;
+
+import java.time.Instant;
+
+@Data
+@Setter(AccessLevel.PRIVATE)
+@EqualsAndHashCode(callSuper = true)
+public class AutomodMessageHoldEvent extends EventSubUserChannelEvent {
+
+    /**
+     * The ID of the message that was flagged by automod.
+     */
+    private String messageId;
+
+    /**
+     * The body of the message.
+     */
+    private Message message;
+
+    /**
+     * The category of the message offense.
+     */
+    private String category;
```

Comment:
I'd like this to be an enum, but twitch doesn't document all of the possible values...